.github/workflows/outdated-dependencies.yml

```diff
@@ -1,37 +1,43 @@
 name: Check outdated dependencies
 on:
-  # TODO: uncomment schedule
-  # schedule:
-  #   # Runs Mon-Fri at 0:00 UTC ( 00:00 EST )
-  #   - cron: "0 0 * * 1-5"
+  schedule:
+    # Runs Mon-Fri at 0:00 UTC ( 00:00 EST )
+    - cron: "0 0 * * 1-5"
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
 jobs:
-  check:
+  check_outdated_dependencies:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [18]
-    # outputs:
-    #   output: ${{ steps.run_outdated.outputs }}
     steps:
       - name: Git checkout
         uses: actions/checkout@v2
-      - name: Use Node.js ${{ matrix.node-version }}
+      - name: Use Node.js 18
         uses: actions/setup-node@v1
         with:
-          node-version: ${{ matrix.node-version }}
-      - id: run_outdated
-        run: npm outdated >> "$GITHUB_OUTPUT"
+          node-version: 18
+      - name: Install dependencies
+        run: npm install
+      - name: Check outdated dependencies
+        run: |
+          echo "$(npm outdated)" > output
+          npm outdated
```

Comment:
way of getting the correct exit status

Reply:
`npm outdated > output` should do this by itself 🤔 Unless your goal was just to always add a newline. Not wrong though

Reply:
I wasn't able to keep the exit status when writing the output into the `output` file, so I'm running the same command again... not pretty but it works